bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Dynamics/Tabulated.cpp

```diff
@@ -96,7 +100,19 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Dynamics_Tabulated(pybind11::module&
 
                     Returns:
                         Frame: The reference frame.
-                
+
+                )doc"
+            )
+
+            .def(
+                "get_interpolation_types",
+                &Tabulated::getInterpolationTypes,
+                R"doc(
+                    Get the interpolation types for each row of the contribution profile.
+
+                    Returns:
+                        list[Interpolator.Type]: The interpolation types.
```

Comment:
It looks like we only set one, so they should all be same? Unless the constructor accepts a list of interpolator types?

Reply:
Good call